microsoftgraph/microsoft-graph-explorer-v4
# Stop using look behind regex

## Overview

Fixes #785 

### Demo

<img width="1328" alt="Screen Shot 2021-01-25 at 3 37 58 PM" src="https://user-images.githubusercontent.com/12011447/105707194-9ef0b000-5f23-11eb-97f4-c0098c3f2f5c.png">

### Notes

I am not sure if this PR introduces a bug but the  url sanitization tests passed. I will need @millicentachieng 👀 on this.

## Testing Instructions

* Start GE
* Open safari and go to localhost:3000
* Observe GE is loaded.
